### ISSUE TITLE
Highliting git output.

### ISSUE THREAD

**Issue by neins** [poucotm/Log-Highlight#28]
Is it possible to highlight git output in sublime panel? It is standart Git plugin from package control.
![Screenshot_1](https://user-images.githubusercontent.com/22091732/60773269-ba128980-a10b-11e9-93b4-f4952449a186.png)

**Comment by poucotm** [comment]
You mean this https://packagecontrol.io/packages/Git ?

**Comment by neins** [comment]
> You mean this https://packagecontrol.io/packages/Git ?

Yes, this is it. On this screenshot: I launched "git status" command from the git plugin.

I expect that when installing Log-Highlight, the syntax in the output will be highlighted. I would like to clarify can i highlight the syntax in the output?

**Comment by poucotm** [comment]
I can see only status bar message or selection box by git : status command. How can I see that below window like you ?

**Comment by poucotm** [comment]
Did you fill out log syntax in settings ? If then, you are able to try to check by right-click and selecting `Log Highlight` command in context menu on that window.  If it is kind of output window and you need `auto highlight`, I can add it to output log window list.

**Comment by neins** [comment]
I just bind these commands by hotkeys like this:

![image](https://user-images.githubusercontent.com/22091732/61174700-604b0d00-a5ac-11e9-9bd0-15e13d5d6a80.png)

And when i call a command by hotkey i got the panel with output:

![image](https://user-images.githubusercontent.com/22091732/61174794-f6336780-a5ad-11e9-9546-09dd33acad2e.png)

About settings, i used default settings and colors and just run "generate custom syntax & theme" in command pallete, and then i click "Log Highlight" by RB on the panel output and got blue-ish background on all these panels (also in case when i call another git command):

![image](https://user-images.githubusercontent.com/22091732/61175004-e79a7f80-a5b0-11e9-9a59-18b5b2a57a28.png)

Unfortunately I discovered another problem, when i restart sublime text or switch project to new window i need click "Log Highlight" on the panel again in order to highlight (got blue bg in my case:) ). By default there is set "auto_highlight" : true.

**Comment by neins** [comment]
Also i cheked output from ShellCommand plugin and there is same behaviour as a git output.
I think it would be nice feature if it possible to manage and highlight output syntax on suchlike plugins :)

**Comment by poucotm** [comment]
![image](https://user-images.githubusercontent.com/19210389/61187202-bf427c00-a6a9-11e9-860a-4cdf348ec2da.png)

I've update the package as v1.9.0.
You can add ""auto_highlight_output_panel": ["exec","git"]," in settings for git output panel.

**Comment by neins** [comment]
Hello, thank you for your message, I've updated plugin and auto highlight work well! 
But unfortunantely there is still no highlighting text, same behaviour as a past times (only blue bg)... 
I was set `"auto_highlight_output_panel": ["exec","git"]` as you advised and this is work for "auto" but not for syntax highlight, then i tried to "erase" and "generate custom syntax & theme" through command pallete and unfortunantely got an error when i try to erase:

![1](https://user-images.githubusercontent.com/22091732/61189249-71227e00-a693-11e9-837c-d3d972650ba6.png)
I got the same error when i try to uninstall plugin, on every stage to reinstall or "erase syntax & theme" i restart sublime and there is no difference. Also i tried manually put `Log Highlight.tmLanguage` loaded from your repo to `User` folder and got the same effect. I read manual few times and don't know what is the problem, may be am i wrong with my settings? There is my file with settings:
[Log Highlight.sublime-settings.txt](https://github.com/poucotm/Log-Highlight/files/3390438/Log.Highlight.sublime-settings.txt)

The main goal to highlight git syntax like in bash terminal - "Monokai" 
Excuse me for so many questions, i appreciate your contributing!
